apps/vaults/contexts/useWalletForExternalMigrations.tsx

```diff
@@ -49,14 +47,13 @@ export const WalletForExternalMigrationsApp = memo(function WalletForExternalMig
 		return tokens;
 	}, []);
 
-	const	{data: balances, update: updateBalances, isLoading, nonce} = useBalances({
+	const {data: balances, update: updateBalances, isLoading, nonce} = useBalances({
 		key: chainID,
-		provider: provider || getProvider(1),
 		tokens: availableTokens,
 		prices
 	});
 
-	const	onRefresh = useCallback(async (): Promise<TDict<TBalanceData>> => {
+	const onRefresh = useCallback(async (): Promise<TDict<TBalanceData>> => {
```

Comment:
We should fix this across the whole codebase :) it's a nice idea to align everything but hard to maintain, so one space is definitely the way to do 👍